src/main/java/ro/unibuc/triplea/domain/auth/model/entity/Game.java

```diff
@@ -0,0 +1,57 @@
+package ro.unibuc.triplea.domain.auth.model.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+
+@Entity
+@Table(name = "Games")
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Game extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(name = "game_steam_id", nullable = false)
+    private Integer gameSteamId;
+
+    @Column(name = "game_name", nullable = true)
+    private String gameName;
+
+    public Integer getId() {
```

Comment:
Instead of generating getter and setters use lombok @Getter and @Setter. Also, you use @Data which contain getter and setters and toString (and others) so all of those can be deleted.

Reply:
fixed